easybuild/framework/easyconfig/constants.py

```diff
@@ -82,3 +82,7 @@ def _get_arch_constant():
     'OS_PKG_PAM_DEV': (('pam-devel', 'libpam0g-dev'),
                        "OS packages providing Pluggable Authentication Module (PAM) developement support"),
 }
+
+# Add EasyConfig constants to export list
+globals().update({name: value for name, (value, _) in EASYCONFIG_CONSTANTS.items()})
+__all__ = ['EXTERNAL_MODULE_MARKER', 'EASYCONFIG_CONSTANTS'] + list(EASYCONFIG_CONSTANTS.keys())
```

Comment:
Hmm, does `__all__.extend(list(EASYCONFIG_CONSTANTS.keys())` work too?

Just to make sure we don't overlooked adding stuff to `all` if additional (real) constants are added to `constants.py`

Reply:
I tested that and no. `__all__` has a default value if it is **not set** (only), so at that point it is an undefined variable.

It actually isn't really required because (almost) nobody does `from easyconfig.constants import *` which is the only use case of `__all__`, everything else works without it.